crates/re_ui/src/command.rs

```diff
@@ -98,7 +98,7 @@ impl UICommand {
             ),
 
             UICommand::ToggleMemoryPanel => (
-                "Toggle memory panel",
+                "Toggle Memory Panel",
                 "Investigate what is using up RAM in Rerun Viewer",
```

Comment:
> Investigate what is using up RAM in Rerun Viewer

This is a loaded choice of words. Can we say something more neutral?

`View and track current RAM usage inside Rerun Viewer`